lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -237,245 +247,61 @@ sub download_asset {
         unlink($assetpath);
         return;
     }
+
     # set proper permissions for downloaded asset
     chmod 0644, $assetpath;
 
 }
 
+sub _remove_if {
+    my ($db, $asset) = @_;
+    return if $asset->{fixed} || $asset->{pending};
+    log_debug "removing " . encode_json($asset);
```

Comment:
Note that the right hand side is always evaluated, also in production mode. So it always does this maybe not completely inexpensive JSON encode.

Reply:
Actually I want this to be log_info - until we know it's working reliable. Then it can go away completely